src/explore-education-statistics-admin/src/pages/release/edit-release/manage-datablocks/components/DataBlockDetailsForm.tsx

```diff
@@ -146,6 +148,11 @@ const DataBlockDetailsForm = ({
                 id="data-block-title"
                 name="title"
                 label="Table title"
+                onChange={e => {
+                  e.preventDefault();
```

Comment:
This wasn't really what I had in mind when I was explaining the code snippet to you yesterday.

We don't need to be preventing the default behaviour here as it should be updating the form.